src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.Custom.pkgdef

```diff
@@ -48,3 +48,9 @@
 "Value"=dword:00000000
 "Title"="Show all C# code actions in Razor files (requires restart)"
 "PreviewPaneChannels"="IntPreview,int.main"
+
+[$RootKey$\FeatureFlags\Razor\LSP\TrimRangesForSemanticTokens]
+"Description"="Trims down ranges when requesting semantic token information on razor files. Note: This is experimental, some actions may not apply correctly, or at all!"
+"Value"=dword:00000000
```

Comment:
Does this value correspond to a default?

Reply:
@davidwengier do you know?

Reply:
Yeah, as far as I know that is the initial value. The `defaultValue` in the C# code is to use when this setting can't be read, which is probably never. This should change to a `1`, the other examples in the file default to off.